configure-places-in-the-sdk/places-extension/using-places-without-active-region-monitoring.md

```diff
@@ -0,0 +1,415 @@
+# Using Places Without Active Region Monitoring
+
+Use cases for your application may not require active region monitoring. Adobe Places can still be used to get your users' location data integrated with other Experience Platform products.
+This document will show how to do a Point of Interest (POI) membership check only at the time of collecting the user's location (latitude and longitude). The document assumes the developer will collect the device's location using the APIs provided by the target platform's operating system.
+
+{% hint style="info" %}
+If your app's use cases require active region monitoring, please read [using the places monitor extension](../places-monitor-extension/using-the-places-monitor-extension.md).
+{% endhint %}
+
+Using Places without active region monitoring can be accomplished in three steps:
+
+1. [Collect the user's location](#collect-the-users-location)
+2. [Retrieve nearby Points of Interest from the SDK based on the new location](#retrieve-nearby-points-of-interest-from-the-sdk)
+3. [Trigger entry events when the user is in a POI](#trigger-entry-events-when-the-user-is-in-a-poi)
+
+## Collect the User's Location
+
+The app developer is responsible for collecting the current location of the device. This can be done via `CoreLocation.framework` (iOS) or the `Location` APIs provided by the `Google Play Services` (Android).
```

Comment:
Please reword as follows:
"The app developer must collect the current location of the device by using the `CoreLocation.framework` (iOS) or the `Location` APIs provided by `Google Play Services` (Android).